jaxrs/src/main/java/com/cerner/beadledom/jaxrs/exceptionmapping/ThrowableExceptionMapper.java

```diff
@@ -0,0 +1,52 @@
+package com.cerner.beadledom.jaxrs.exceptionmapping;
+
+import static javax.ws.rs.core.Response.Status.INTERNAL_SERVER_ERROR;
+
+import com.cerner.beadledom.json.common.model.JsonError;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.ext.ExceptionMapper;
+import javax.ws.rs.ext.Provider;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * An {@link ExceptionMapper} for the {@link Throwable} family of exceptions.
+ *
+ * <p>The intention of this exception mapper is to handle all unhandled exceptions thrown by a
+ * service (i.e. exceptions that are not handled in try/catch blocks or do not have an
+ * {@link ExceptionMapper} already implemented). The exceptions are restructured into a standard
+ * JSON format.
+ *
+ * @author Brian van de Boogaard
+ * @author Cal Fisher
+ * @since 2.6
+ */
+@Provider
+public class ThrowableExceptionMapper implements ExceptionMapper<Throwable> {
+
+  private static final Logger logger = LoggerFactory.getLogger(ThrowableExceptionMapper.class);
+
+  /**
+   * Maps an unhandled {@link Throwable} to a {@link Response}.
+   *
+   * @param exception the {@link Throwable} exception that was not handled.
```

Comment:
nit: no periods at the end of at clauses. Periods should only be in the summary sentence and description paragraphs.

Reply:
Removed periods from the end of at clauses: https://github.com/cerner/beadledom/pull/45/commits/cf92faf55674bfa402abe51eff5459464d3ba663#diff-ef962290f5cec93a2a13419308fdf78aR33